programs/librapay/src/librapay_transaction.rs

```diff
@@ -161,16 +161,8 @@ mod tests {
     fn create_bank(lamports: u64) -> (Arc<Bank>, Keypair, Keypair, Pubkey, Pubkey) {
         let (mut genesis_config, mint) = create_genesis_config(lamports);
         genesis_config.rent.lamports_per_byte_year = 0;
-<<<<<<< HEAD
```

Comment:
@CriesofCarrots Merge conflicts?

Reply:
Right, I'm removing those. Those went into the code base here (https://github.com/solana-labs/solana/pull/9818) and here (https://github.com/solana-labs/solana/pull/9814)

Reply:
Heh, thanks, I need to re-enable libra sooner than later to avoid this kind of bit rot

Reply:
I can remove them on master

Reply:
I don't think they are on master

Reply:
Nevermind, just got put in mergify on v1.1